### PR TITLE
PMM-6827 DB cluster monitoring.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**
[PMM-6827](https://jira.percona.com/browse/PMM-6827)

- [x] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1239

- [ ] Links to other linked pull requests (optional).
---
- [ ] Tests passed.
- [ ] Feature build pass.
- [ ] (Re)requested review.
- [ ] Fix conflicts with target branch.
- [ ] Update API dependency.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

### CODE REVIEW

File: models/settings.go [hunk 1]
```diff
@@ -71,9 +76,29 @@ type Settings struct {
 	} `json:"dbaas"`
 }
 
+func getPublicIP() (string, error) {
```
Comment (BupycHuk) [inline models/settings.go:79]: it's out of scope.

**Comment by JiriCtvrtka** [comment]
Build failed on error which is not presented in my local env. Working on fix.

**Review by percona-csalguero — APPROVED** [review]

**Review by askomorokhov — APPROVED** [review]

File: Gopkg.toml [hunk 2]
```diff
@@ -13,15 +13,15 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
+  branch = "PMM-6827-db-cluster-monitoring"
```
Comment (askomorokhov) [inline Gopkg.toml:16]: 👀

File: Gopkg.toml [hunk 3]
```diff
@@ -13,15 +13,15 @@ required = [
...
 
 [[constraint]]
   name = "github.com/percona-platform/saas"
   branch = "main"
 
 [[constraint]]
   name = "github.com/percona-platform/dbaas-api"
-  branch = "main"
+  branch = "PMM-6827-db-cluster-monitoring"
```
Comment (askomorokhov) [inline Gopkg.toml:24]: 👀

**Pull request merged**